src/main/java/com/superdevs/HealthOMeter/entity/HistoricalMedicalData.java

```diff
@@ -11,42 +11,43 @@
     @Column(name = "id")
     private long id;
 
+    @Column(name = "created")
+    private LocalDateTime created;
+
     @ManyToOne
     @JoinColumn(name = "contact_id")
     private Contact contact;
 
-    @Column(name = "createde")
-    private LocalDateTime createde;
-
     public HistoricalMedicalData() {
     }
 
-    public HistoricalMedicalData(Contact contact) {
+    public HistoricalMedicalData(long id, Contact contact) {
+        this.id = id;
+        this.created = LocalDateTime.now();
```

Comment:
This one concerns me a little. I wonder will this update HistoricalMedicalData.created when we will move them from MedicalData to HistoricalMedicalData.... @Nebres84 , what are your thoughts?